### PR TITLE
Add date input component

### LINKED ISSUE #302: ADD DATE-TIME INPUT FOR START DATE
- [ ] create date-time input component
- [ ] store input value

### PR DESCRIPTION
**Pull request by hippiedev**
Closes #302
I created a date input component and rendered it to the DOM.

To find it:

On [goals.zuri.chat](url), click "open modal" (Green conspicuous button on the screen)
Scroll to "Date for goal completion", below you will see the input component for adding date.

### CODE REVIEW

File: client/src/components/createGoal/GoalForm.js [hunk 1]
```diff
@@ -125,16 +126,7 @@ const GoalForm = React.forwardRef((props) => {
...
-              <Input
-                type="text"
-                id="date"
-                onFocus={(e) => {
-                  e.currentTarget.type = 'date';
-                }}
-                // eslint-disable-next-line no-return-assign
-                onBlur={(e) => (e.currentTarget.type = 'text')}
-                placeholder="End Date"
-              />
```
Comment (abiona01) [inline client/src/components/createGoal/GoalForm.js:137]: Does the Placeholder still show Date after you deleted this block of code?

**Review by abiona01 — CHANGES_REQUESTED** [review]

**Comment by Asylcreek** [comment]
Hey so, you still have conflicts, pleaes pull from the staging branch, resolve them conflicts and make the PR again

File: client/src/components/createGoal/GoalForm.js (hunk 1 above)
Comment (hippiedev) [reply client/src/components/createGoal/GoalForm.js:137]: Yes, but that would take longer to do. But I will work on it.

**Comment by hippiedev** [comment]
> Hey so, you still have conflicts, pleaes pull from the staging branch, resolve them conflicts and make the PR again

Conflicts resolved

**Comment by abiona01** [comment]
> > Hey so, you still have conflicts, pleaes pull from the staging branch, resolve them conflicts and make the PR again
> 
> Conflicts resolved

You're yet to answer my comment. Does the date input component display a placeholder like it does on the design? If it doesn't kindly revert the date input back to the way it was. Also, this will not pass for review. The whole createGoal component has been reviewed.

**Comment by hippiedev** [comment]
> > > Hey so, you still have conflicts, pleaes pull from the staging branch, resolve them conflicts and make the PR again
> > 
> > 
> > Conflicts resolved
> 
> You're yet to answer my comment. Does the date input component display a placeholder like it does on the design? If it doesn't kindly revert the date input back to the way it was. Also, this will not pass for review. The whole createGoal component has been reviewed.

Okay

**Pull request closed without merging**